website/docs/getting-started/setup/installation-guides/docker/README.mdx

```diff
@@ -0,0 +1,199 @@
+
+<VideoEmbed host="youtube" videoId="Tde7GqE6FQQ" /> 
+
+## Prerequisites
+
+* [Docker](https://docs.docker.com/get-docker/) (version 20.10.7 or later)
+* [Docker-Compose](https://docs.docker.com/compose/install/) (version 1.29.2 or later) _(Optional)_
+
+1. Create an installation folder called `appsmith` where you would like your Appsmith installation and data storage.
+2. `cd` into the installation folder.
+
+## Setup with Docker Compose _(recommended)_
+
+The Appsmith Docker image contains all the components required to run, within a single Docker container. All these multiple processes are managed by a [Supervisord](http://supervisord.org/) instance, which is a lightweight process manager.
+
+<Tabs groupId="editions" >
+   <TabItem label="Community Edition" value="community">
+
+1. Click to [download the `docker-compose.yml` file](/img/docker-compose_(3).yml), and place it into your Appsmith installation folder.
+
+**OR,** run the following cURL:
+
+```bash
+curl -L https://bit.ly/3AQzII6 -o $PWD/docker-compose.yml
+```
+
+2. Start the Docker container by running the command below. You may need to run with `sudo` if `docker` and `docker-compose` aren't accessible by your user. If the image doesn't exist locally, this command downloads the necessary Docker image and starts the container.
+
+```bash
+docker-compose up -d
+```
+
+   </TabItem>
+   <TabItem label="Business Edition" value="business">
+
+:::note
+To upgrade your existing Community Edition installation to the Business Edition, [follow these instructions](/getting-started/setup/upgrade-to-business-edition/docker).
+:::
+
+1. Click to [download the `docker-compose.yml` file](/img/docker-compose_(3).yml), and place it into your Appsmith installation folder.
+
+**OR,** run the following cURL:
+
+```bash
+curl -L https://bit.ly/3AQzII6 -o $PWD/docker-compose.yml
+```
+
+2. Change the **image** name from `appsmith-ce` to `appsmith-ee` for the `image:` key in the `docker-compose.yml` file.
+
+3. Add your **license key** (`APPSMITH_LICENSE_KEY`) to the `environment:` key in the `docker-compose.yml` file.
+
+4. Start the Docker container by running the command below. You may need to run with `sudo` if `docker` and `docker-compose` aren't accessible by your user. If the image doesn't exist locally, this command downloads the necessary Docker image and starts the container.
+
+```bash
+docker-compose up -d
+```
+
+   </TabItem>
+</Tabs>
+
+Your Appsmith server should be up and running soon. You can access it at [http://localhost](http://localhost). You can also verify that it's working correctly in the [Docker logs](#docker-logs).
+
+If this is your first time setting up using Docker, you should expect a welcome page similar to the one below.
+
+![Welcome Page](</img/image_(1)_(1).png>)
+
+### Updating Appsmith
+
+To update Appsmith (configured with docker-compose) manually, go to the root directory of your installation and run the following command:
```

Comment:
is 'configured with docker-compose' necessary, we have this under that section itself right?
Also replace 'your' with 'the'